omod/src/main/webapp/index.htm

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
```

Comment:
This file is worth reviewing. It is the index.html that will be the SPA for all open mrs code.

It's very barebones and has nothing in it yet, but subsequent PRs will add to it.